extensions/adobe/experience/offer-management/rank.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/rank",
+  "type": "object",
+  "title": "Offer Rank",
+  "description": "Offer rank (priority)",
+  "definitions": {
+    "rank": {
+      "properties": {
+        "xdm:priority": {
```

Comment:
What does this number mean?  Is it an integer or a float?  Does it have a min or max?

Reply:
updated

Reply:
👍